### PR TITLE
Resolves some issues, gim chat and archive history count setting

### PR DESCRIPTION
**Pull request by sololegends**
- Adds toggle for saving group iron man chat channel
- Adds toggle for saving game messages channel
- Added configuration value for max historical archives on chat logs
- Added ability to store logs in a folder per osrs user

Resolves issue #13
Resolves issue #9

### CODE REVIEW

**Comment by sololegends** [comment]
@hex-agon You made an edit to this PR to mark this as an unmaintained repo. 

I made a [PR on runelite plugin-hub](https://github.com/runelite/plugin-hub/pull/5612) about an updated, added the things in this pr + some, slightly cut down (no remote) replacement.    

They mentioned to ask you about becoming a maintainer for this plugin, so that users of this plugin can get updates as opposed to a new plugin being added.  

TL;DR: If you don't want to maintain this anymore, can I be given access to maintain it in your stead.

**Comment by hex-agon** [comment]
i invited you as a collaborator
